src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
```

Comment:
We also need to discuss having Prettier installed. Some of us have it installed and it changes something in the code alignment. For example, when I fetch this branch. Most of the alignment is changed. Am saying this, because this could be an issue in the future when someone pushes code and changes files that aren't supposed to be touched. More so, those who have Prettier installed and have their VS Code set to auto save, won't see the changes Prettier makes until they've done CMD+S. Again, I would like us to have a conversation on this.

Reply:
Good point @missvicki. However, in my opinion, I would suggest we stick to only the `eslint` style rules for two reasons.
- `eslint` displays style warnings but does not go ahead and reformat code without you explicitly doing it. `Prettier` on the other hand goes ahead and auto reformats, which as you have observed sometimes leads to changes to none relevant parts of code.
- I believe the eslint rules will suffice for the project.
We could discuss this further though.

Reply:
Currently, there are more than 4 spaces that it is giving off and that looks so weird. Could this be changed by defining the tab space